src/sidebar/components/Annotation/AnnotationBody.js

```diff
@@ -64,13 +64,13 @@ function AnnotationBody({ annotation, settings }) {
             markdown={text}
             textClass={{
               AnnotationBody__text: true,
-              'is-hidden': isHidden(annotation),
+              'p-redacted-content': isHidden(annotation),
```

Comment:
Instead of defining an `is-hidden` class for this local component, and using it to wrap a mixin, I think it's a little easier to reason about `p-redacted-content` directly here, as it better expresses what's actually happening.